src/redux/constants/_initialState.js

```diff
@@ -4,4 +4,118 @@ export const initialState = {
     isOpen: false,
     type: "",
   },
+  postings: [
+    {
+      id: 0,
+      title: "Game Boy Advanced SP",
+      datePosted: "2018-07-01T00:00:00.000Z",
+      description:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus eu est volutpat, luctus mauris id, bibendum sem. Mauris ultrices diam eu metus mattis, ut sodales tellus rhoncus. Nunc eleifend rutrum tortor, sit amet mattis mi euismod ac. Mauris diam ante, tincidunt in mauris et, ullamcorper fermentum urna.",
+      category: "electronics",
+      condition: "good",
+      quantity: 1,
+      tags: ["kid friendly", "retro gaming"],
+      requestedItems: [
+        "DualShock4 Controller",
+        "Nintendo Switch Joy-Cons",
+        "Hand Sanitizer",
+        "Baked Goods",
+      ],
+      images: ["gameboy.jpg"],
+      ownerId: "2131254",
+      ownerUsername: "hulkbuster",
+      postalCode: "V73P6G1",
+      offerings: [],
+      location: "North Vancouver, BC",
```

Comment:
Are we going to have both Postal Code and Location?

Reply:
**Follow up**: Item Preview uses Location, so to prevent breaking things, location is included.